inject/service/pom.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2023 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="http://maven.apache.org/POM/4.0.0"
+        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>io.helidon.inject</groupId>
+        <artifactId>helidon-inject-project</artifactId>
+        <version>4.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>helidon-inject-service</artifactId>
```

Comment:
maybe name this `inject-service-api`? Just feels weird to have inject-api depend on inject-service. Ideally it would have been reversed, right.

Reply:
I still think `inject-api` should go away, so we would have: `inject-service` and `inject`.
I do not want to name this module `inject-api`, as it would seem to be API of the Helidon Injection, which is not correct (as that is `Service` and related types). If we come up with better name than service, we can use it

Reply:
Closing this conversation. We still have time to re-open it, as long as the branch remains open